src/hooks/useOpenAiSSEResponse.ts

```diff
@@ -66,7 +67,14 @@ const useOpenAiSSEResponse = ({
       let textContent = "";
       if (type === "article" || type === "song") {
         const json = await fetchArticleData(url, 500);
-        mappedResult.current = { ...mappedResult.current, byline: json.byline, title: json.title, dir: json.dir, url };
+        mappedResult.current = {
+          ...mappedResult.current,
+          byline: json.byline,
+          title: json.title,
+          dir: json.dir,
+          url,
+          // content: json.content,
```

Comment:
if we uncomment this the content will pass through, but currently the url encode encodes the content which leads to urls that don't work.

Reply:
@MaiaS the song seems to be working fine, but the articles doesn't. 🤔 any idea what could be the issue? i'll check this later in the afternoon